pyphare/pyphare/pharesee/plotting.py

```diff
@@ -60,7 +61,19 @@ def dist_plot(particles, **kwargs):
               bins=kwargs.get("bins", bins),
               weights=particles.weights)
 
-    sig = kwargs.get("sigma", (0,0))
+    if "gaussian_filter_sigma" in kwargs: 
+        from scipy.ndimage import gaussian_filter
+        sig = kwargs.get("gaussian_filter_sigma", (0,0))
+        image = gaussian_filter(h.T, sigma=sig)
+    elif "median_filter_size" in kwargs: 
```

Comment:
what if the user passes both gaussian_filter_sigma and median_filter_size?

Reply:
good remark... now it is clear that the 2 filters can not be called at the same time